pkg/binding/bindingpolicy.go

```diff
@@ -279,10 +279,11 @@ type mrObject interface {
 // testObject tests if the object matches the given tests.
 // The returned tuple is:
 //   - bool: whether the object matches ANY of the tests
+//   - bool: whether any of the tests says CreateOnly==true
 //   - sets.Set[string]: the UNION of the statuscollector names that appear within
 //     EACH of the tests that the object matches
 func (c *Controller) testObject(ctx context.Context, objIdentifier util.ObjectIdentifier, objLabels map[string]string,
```

Comment:
Perhaps now is a good time to move this function.

Reply:
Let's do that in a separate PR.